threescale_api/defaults.py

```diff
@@ -53,6 +53,10 @@ def parent(self) -> 'DefaultResource':
         """
         return self._parent
 
+    @parent.setter
+    def parent(self, parent):
```

Comment:
better to rename self._parent to self.parent, remove the `@property`

Reply:
done